dts/bindings/spi/ti,mspm0-spi.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2024 Bang & Olufsen A/S, Denmark
+# SPDX-License-Identifier: Apache-2.0
+
+description: TI MSPM0 SPI controller
+
+compatible: "ti,mspm0-spi"
+
+include: [spi-controller.yaml, pinctrl-device.yaml]
+
+properties:
+  reg:
+    required: true
+
+  pinctrl-0:
+    required: true
```

Comment:
Should pinctrl-0 be required?

Reply:
You can't use the hardware if the pinctrl is not setup as SPI